device/esp_tinyusb/Kconfig

```diff
@@ -11,10 +11,14 @@ menu "TinyUSB Stack"
         prompt "TinyUSB PHY"
         default TINYUSB_RHPORT_HS
         help
-            Allows set the USB PHY Controller for TinyUSB: HS (USB OTG2.0 PHY for HighSpeed)
+            Allows set the USB PHY Controller for TinyUSB:
+            - HS (USB OTG2.0 PHY for HighSpeed)
+            - FS (USB OTG1.1 PHY for FullSpeed)
 
         config TINYUSB_RHPORT_HS
             bool "HS"
+        config TINYUSB_RHPORT_FS
+            bool "FS"
```

Comment:
If possible, I'd like the peripheral selection to be runtime configurable from the very first release.

Could you please investigate this option more? IIRC, it is possible only from our tinyusb v0.17.0~1 onwards

Reply:
Yeah, sure.
This PR just was already there, so I thought we could use it, provide the configurable option and then change it to dynamically configurable feature. I’ll check.

Reply:
BTW, this will not work without the PHY changes in esp-idf, right? 

If we need the esp-idf changes, then we can postpone this feature to v1.7.0.... Up to you :)